### DISCUSSION TITLE
FieldLabel RequiredIndicator not shown in Tailwind

### DISCUSSION THREAD

**Discussion by mdmontesinos** [Megabit/Blazorise#5119] [General]
Fields marked with RequiredIndicator do not display the '*' when using Tailwind.

Here's the proof from the Blazorise demos:

Boostrap version:

![image](https://github.com/Megabit/Blazorise/assets/90258222/300c5fda-7e82-4ac1-8919-126a9fa04f46)

Tailwind version:

![image](https://github.com/Megabit/Blazorise/assets/90258222/12e59731-58f0-4259-939a-98c496b08b3b)

Although the tailwind version does include the after:content-[' *']

```html
<label class="block my-auto text-sm font-medium text-gray-900 dark:text-white after:content-[' *'] after:[color:var(--b-theme-danger, --btw-color-danger-500)] relative w-full md:basis-full relative w-full lg:basis-3/12 pl-2 pr-2">Full Name</label>
```

**Comment by stsrki** [comment]
Seems like a bug. I will create an issue.

**Comment by mdmontesinos** [comment]
Resolved via #5123.

Thanks!
